extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -170,6 +170,7 @@ export class AmpAdXOriginIframeHandler {
             this.element_.warnOnMissingOverflow = false;
           }
           this.handleResize_(
+            data['id'],
```

Comment:
I'm not very familiar with this code. Can you explain when this has an id or not, and why there is no id for `renderStartMsgHandler_`?

Reply:
`renderStartMsgHandler_` would send a resize_success event upon RENDER_START, therefore it is not associated with any resize requests and no resize request ID is available.

Reply:
Got it. Thanks!